onnxruntime/core/providers/nnapi/nnapi_builtin/builders/helper.h

```diff
@@ -29,6 +29,27 @@
             std::string(", ") + (note));                                  \
   }
 
+#define RETURN_STATUS_ON_ERROR(val)                                       \
+  {                                                                       \
+    const auto ret = (val);                                               \
+    ORT_RETURN_IF_NOT(                                                    \
+        ret == ANEURALNETWORKS_NO_ERROR,                                  \
+        std::string("Error in ") + __FILE__ + std::string(":") +          \
+            std::to_string(__LINE__) + std::string(", function name: ") + \
+            std::string(__func__) + "error, ret: " + GetErrorCause(ret)); \
```

Comment:
ORT_WHERE could be used

Reply:
ORT_WHERE is already part of ORT_RETURN_IF_NOT, removed these redundant strings

---
In reply to: [462089796](https://github.com/microsoft/onnxruntime/pull/4650#discussion_r462089796) [](ancestors = 462089796)